school-finder-frontend/package.json

```diff
@@ -9,12 +9,23 @@
   "license": "MIT",
   "devDependencies": {
     "angular-leaflet-directive": "^0.7.11",
+    "angular-mocks": "^1.3.11",
     "angular-ui-router": "^0.2.13",
     "chai": "^1.10.0",
     "gulp": "^3.8.10",
     "gulp-mocha": "^2.0.0",
+    "karma": "^0.12.31",
+    "karma-chai": "^0.1.0",
+    "karma-chai-plugins": "^0.2.4",
+    "karma-chrome-launcher": "^0.1.7",
+    "karma-firefox-launcher": "^0.1.4",
+    "karma-ie-launcher": "^0.1.5",
+    "karma-mocha": "^0.1.10",
+    "karma-nyan-reporter": "0.0.50",
     "leaflet": "^0.7.3",
-    "mocha": "^2.1.0"
+    "mocha": "^2.1.0",
+    "phantomjs": "^1.9.15",
```

Comment:
I'd argue we don't test this code in the browser at all.
We can stub out the geolocation calls as we know the various outcomes and just do it that way. 
Makes the tests a lot easier and cuts down on the complexity of our testing procedures.